content/diversity-accessibility.html

```diff
@@ -87,5 +87,11 @@
 <h2>Quiet room</h2>
 <p>We will provide a clearly-marked designated quiet room at all venues. This room is intended to be a calm and quiet place for anyone who needs to have a break from the bustle of the conference, and will not be used for socialising.</p>
 
+<h2>Clothing</h2>
+<p>At PyCon UK we want everyone to feel comfortable, and that includes wearing clothing that makes you feel comfortable. For that reason, PyCon UK <strong>does not have a formal dress code</strong>. As long as your outfit is not illegal and does not violate the <a href="/code-of-conduct">Code of Conduct</a>, <strong>you are welcome to wear whatever suits you best</strong>, whether that is national costume, religious dress, business suits, or any other preferred form of dress or decoration.</p>
+
+<h2>Faith</h2>
+<p>PyCon UK is fortunate to have delegates from all walks of life, including delegates from many different faith backgrounds. While Pycon UK itself is a secular event, it is extremely important to us that our delegates are able to satisfy their spiritual needs as well as their physical ones. Happily, Cardiff is a multicultural city with places of worship for many different faiths. If you are in search of an appropriate place of worship, a quick internet search will locate one: alternatively, if you need more assistance, <a href="http://www.visitcardiff.com/discover/touristinformationcentre/">Cardiff tourist information</a> will gladly help you find what you need.</p>
```

Comment:
"While Pycon UK itself.." needs to be "While PyCon UK itself..."

Never seen a test like that, but can totally see why `:)`